atd-vze/src/queries/Locations.js

```diff
@@ -63,8 +63,8 @@ export const UPDATE_LOCATION_POLYGON = gql`
 export const locationQueryExportFields = `
 location_id
 description
-crashes_count_cost_summary { total_crashes }
-crashes_count_cost_summary { total_deaths }
-crashes_count_cost_summary { total_serious_injuries }
-crashes_count_cost_summary { est_comp_cost }
+crash_count
+fatalities_count
+serious_injury_count
+total_est_comp_cost
```

Comment:
Seems like we dont need a relationship here at all because these columns already exist in the view that the Locations GridTable uses to display the data, so I just put those columns here. Very curious about the context of these different views though if anyone knows

Reply:
@roseeichelmann This was a great catch, and thanks for fixing it! I'm about to review this, but, to give some context, I think this was a regression that came out of the changes in [this PR](https://github.com/cityofaustin/atd-vz-data/pull/1196). I should have tested the export before I approved that one. 😅

Reply:
Ohh yup thats it, thanks! I should have tested that too!